quarantine-status/stacks/lambda.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
One of the JSON or YAML versions of the CFN is sufficient. I'd vote for keeping the YAML version.